### PR TITLE
Changed ticker name. Updated balances and keys. Introduced inflation curve for era payouts.

### PR DESCRIPTION
**Pull request by DanieleDiBenedetto**
As per title.
Inflation curve parameters have been taken from Polkadot, we'd need to customize them according to our needs in future releases.
All the code and tests related to the simulation of ZEND payouts have been removed.
CDK aggregator address has been removed as well from the chainspec.
Ticker name has been changed to ZETA.
Balances have been updated to sum to around 1B

### CODE REVIEW

File: runtime/src/tests.rs [hunk 1]
```diff
@@ -560,43 +560,36 @@ mod pallets_interact {
...
 
-    use crate::{Balance, Runtime, CENTS};
+    use crate::{Balance, Runtime};
 
     use super::new_test_ext;
 
-    /// Test that validators receive a cumulative reward that mimics the current emission of
-    /// $ZEN in the PoW Horizen blockchain for miners, which is a coinbase of 6.25 Zen for
-    /// each block every 2.5 minutes.
+    #[ignore]
```
Comment (JackPiri) [inline runtime/src/tests.rs:567]: Maybe it's better to remove the `ignore`.

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

File: runtime/src/tests.rs (hunk 1 above)
Comment (DanieleDiBenedetto) [reply runtime/src/tests.rs]: Done thanks

**Review by la10736 — APPROVED** [review]
LGTM

**Pull request merged**